src/components/StatusDashboard/current_migrations.jsx

```diff
@@ -99,7 +99,7 @@ function TableContent({ collapsed, name, resort, rows, select, sort }) {
     <>
       <thead>
         <tr onClick={select}>
-          <th colSpan={7} className={collapsed ? styles.collapsed : undefined}>
+          <th colSpan={8} className={collapsed ? styles.collapsed : undefined}>
```

Comment:
Thanks Sylvain! 🙏

Is this what we are looking for?

Reply:
I think so.